front/src/modules/timesStops/helpers/utils.ts

```diff
@@ -213,6 +217,21 @@ export function updateRowTimesAndMargin(
   return newRowData;
 }
 
+/**
+ * This function is called before comparing rows to prevent a change from undefined to null (or the reverse)
+ * from being treated as an actual update of a row (otherwise changes would occur on deletion of an undefined field)
+ */
+export function normalizeNullablesInRow(row: TimesStopsInputRow): TimesStopsInputRow {
+  const normalizedRow = _.clone(row);
```

Comment:
Nit: I'd slightly prefer `= { ...row }` because it's standard JavaScript instead of relying on an external library.

Reply:
Sure, I agree!